apps/i18n/common/en_us.json

```diff
@@ -1417,8 +1417,8 @@
   "makerPageTitle": "Code.org Maker App",
   "makerSetupDeprecationWarningAppTitle": "Maker App is being deprecated June 2023",
   "makerSetupDeprecationWarningAppDetails": "Maker Toolkit no longer requires the Maker App to connect to boards. After June 2023, Code.org will remove support for the Maker App. To learn more, please refer to",
-  "makerSetupDeprecationWarningOldChromeTitle": "Maker Toolkit will require a new version of Chrome",
-  "makerSetupDeprecationWarningOldChromeDetails": "After June 2023, the Maker Toolkit will not support your current version of Chrome. Please update to a version higher than 90 before June 2023 to continue using the Maker Toolkit. To learn more, please refer to",
+  "makerSetupDeprecationNoticeOldChromeTitle": "Maker Toolkit requires a newer version of Chrome",
+  "makerSetupDeprecationNoticeOldChromeDetails": "The Maker Toolkit does not support your current version of Chrome. Please update to a version greater than or equal to 90 to continue using the Maker Toolkit. To learn more, please refer to",
```

Comment:
Minor - not sure how likely it is that we'll ever update the minimum Chrome version requirement, but we could maybe pass the version in to this string rather than hardcoding it here, so we're reusing the same value in both places (for example, the MIN_CHROME_VERSION constant that @thomasoniii suggested)?